api/client/cloud/cloud_test.go

```diff
@@ -811,28 +811,15 @@ func (s *cloudSuite) TestUpdateCloudsCredentialsManyMatchingResults(c *gc.C) {
 	ctrl := gomock.NewController(c)
 	defer ctrl.Finish()
 
-	args := params.UpdateCredentialArgs{
-		Force: false,
-	}
 	count := 2
-	for tag, credential := range createCredentials(count) {
-		args.Credentials = append(args.Credentials, params.TaggedCredential{
-			Tag: tag,
-			Credential: params.CloudCredential{
-				AuthType:   string(credential.AuthType()),
-				Attributes: credential.Attributes(),
-			},
-		})
-	}
-
 	res := new(params.UpdateCredentialResults)
 	results := params.UpdateCredentialResults{
 		Results: []params.UpdateCredentialResult{
 			{},
 			{},
 		}}
 	mockFacadeCaller := basemocks.NewMockFacadeCaller(ctrl)
-	mockFacadeCaller.EXPECT().FacadeCall("UpdateCredentialsCheckModels", args, res).SetArg(2, results).Return(nil)
+	mockFacadeCaller.EXPECT().FacadeCall("UpdateCredentialsCheckModels", gomock.Any(), res).SetArg(2, results).Return(nil)
```

Comment:
this removes most of the content of this test. suggest you using a go mock matcher instead

Reply:
Oh yeah thanks this might be better. Putting one right away.